src/pages/patientView/mutation/style.scss

```diff
@@ -0,0 +1,5 @@
+[data-test=LazyMobXTable] {
```

Comment:
the data-test tags are really just used for testing, I don't think they should be mixed with business logic/styling

Reply:
Good point. I have introduced a new class on the table container and referenced that in the css.